srcpkgs/j/template

```diff
@@ -98,7 +104,7 @@ else
 	exit 1
 fi
 
-${_jlib}/${_jsz}\${javx}/jconsole -jprofile "${_jetc}/profile.ijs" "\${@}"
+exec ${_jlib}/${_jsz}\${javx}/jconsole -jprofile "${_jetc}/profile.ijs" "\${@}"
```

Comment:
Why do we need `exec`?

Reply:
It's just a launcher script for jconsole repl, so philosophically, we don't want to keep the shell process around. Practically, though, I made the change so that tmux can pick up on the process name. Without `exec` the best tmux can do is set `pane_current_command` to the `sh` wrapper, which is annoying.